recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: 9706ffa858a98bb1b97162e3887aa03c6c746017a274951e94a6341a59efad12
 
 build:
-  number: 2
+  number: 3
```

Comment:
By the way, I don't think a version bump is necessary here? Since we're just adding a migration, should add a 'Skip CI' message perhaps.